cmd/devp2p/dns_route53.go

```diff
@@ -109,21 +113,35 @@ func (c *route53Client) deploy(name string, t *dnsdisc.Tree) error {
 	batches := splitChanges(changes, route53ChangeSizeLimit, route53ChangeCountLimit)
 	for i, changes := range batches {
```

Comment:
this loop can be optimized, mostly because the waiter is blocking (this is the case in the current version as well as in this PR). 

Currently for every batch, it makes the change and then waits up to 30 minutes for the change to propagate before making the next change for the next batch. In a worse case scenario, this loop could block a command for hours.

My proposal is that either
1. we loop through the batch and make all of the changes, then block the thread until all changes are made. This way the bottleneck only exists on the AWS side of how fast changes are propagated

2. create a go-routine for each batch. This way each routine is responsible for making the change and then waiting for the propagation. The thread would lock until all of the routines have broken out of the wait loop.

option #2 would be more elegant, both options would be a dramatic optimization. However I am not clear if this is necessary or desired. If it is I can include those changes in this PR or a different one (I can also create the optimization with the v1 sdk)

Reply:
If you want to make this change, I prefer option 1. There can be a lot of changes, and starting a goroutine for each has its own problems.

Reply:
@fjl I have implemented option #1. Propagation takes an average of 60 seconds, according to AWS, so now instead of waiting ~ 60-90 seconds per change, ideally it would take 60-90 seconds for the first change and then the rest of the waiters should be successful on the first request